CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+# 1.0.1
+
+* Bug fix: Initial requests preferred earlier peers. Shuffle peers on
+* Peers can now be removed using PeerList.Remove
+* Add ErrorHandlerFunc to create raw handlers that return errors
+* Retries now avoid hosts (not just host:pair)
+  PeerList.Add.
+* Create ArgReader (an alias for io.ReadCloser) for symmetry with ArgWriter.
+* Unify ArgReadable and ArgWritable interfaces out of the raw and http
+  sub-packages
+* Expose Thrift binary encoding methods
+  (ReadStruct, WriteStruct, ReadHeaders, WriteHeaders)
```

Comment:
This doesn't seem very human-readable to me - is the first item a sentence? A manual editing pass would be helpful.